src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -145,6 +145,13 @@ public partial class AuthenticationResult
                     tenantProfiles: tenantProfiles);
             }
 
+            UniqueId = msalIdTokenCacheItem?.IdToken?.GetUniqueId();
```

Comment:
Minor: msalIdTokenCacheItem null is checked in 3 places and idToken in 2 places. It will not make much perf difference, but if you want it can be done this way.
if(msalIdTokenCacheItem != null)
{
IdToken = msalIdTokenCacheItem.Secret;
// you can use if(msalIdTokenCacheItem.IdToken)
UniqueId = msalIdTokenCacheItem.IdToken?.GetUniqueId();
TenantId = msalIdTokenCacheItem.IdToken?.TenantId;
}